tests/integration/test_data/private_repo_packages.yaml

```diff
@@ -1,11 +1,17 @@
 # Test data for the private repos test
 # The test will only be performed if $JOB_NAME includes one of the following
+
 private_repo_test_envs:
   - cachito-prod
   - cachito-stage
 private_repo_https:
   repo: https://github.com/cachito-testing/cachito-no-package-manager-private.git
-  ref: 31e242befe367f940f878f27158d9cd036ba3b23
 private_repo_ssh:
   repo: git@github.com:cachito-testing/cachito-no-package-manager-private.git
-  ref: 31e242befe367f940f878f27158d9cd036ba3b23
+
+# repo url for commit creation
+private_repo_url: git@github.com:cachito-testing/cachito-no-package-manager-private.git
+# test repo user
+git_user: "Arthur Dent"
+# test repo user email
+git_email: "dent42@cachito.rocks"
```

Comment:
Just a heads-up that, since the file now contains environment-specific data, the private_repo_packages.yaml file will need to be duplicated in our private gitlab :(

https://gitlab.cee.redhat.com/osbs/cachito-tests

Reply:
Or maybe we could move git_user and git_email out of the `test_data/*.yaml` files and put it in `test_env_vars.yaml` instead.

Which would also require a change in the private repo, but only in test_env_vars.yaml. We wouldn't have to duplicate yet another test_data file

Reply:
Cool! https://gitlab.cee.redhat.com/osbs/cachito-tests/-/merge_requests/82 link for PR

Reply:
I would rather move git vars to `test_env_vars.yaml` too.

Reply:
+1 to use `test_env_vars.yaml` and avoid creating additional test_data file